Tests/FunctionalTests/Sources/FTRVisibilityTest.m

```diff
@@ -41,25 +40,26 @@ - (void)testOverlappingViews {
   [[EarlGrey selectElementWithMatcher:grey_accessibilityID(@"bottomScrollView")]
       performAction:grey_scrollToContentEdge(kGREYContentEdgeTop)];
 
+  BOOL (^assertionBlock)(id element, NSError *__strong *errorOrNil) =
+      ^BOOL(id element, NSError *__strong *errorOrNil) {
+         CGPoint offset = ((UIScrollView*) element).contentOffset;
+         CGPoint expectedOffset = CGPointMake(100, 100);
+         if (CGPointEqualToPoint(offset, expectedOffset)) {
+           return YES;
+         } else {
+           NSError* error = [[NSError alloc]
+                             initWithDomain:kGREYInteractionErrorDomain
```

Comment:
Is this formatted right?

Reply:
I think so. The transform script did not change any format I think.

Reply:
let's fix this format here, also combine this with the line below.